packages/contracts/src/EmailWalletCore.sol

```diff
@@ -547,9 +554,15 @@ contract EmailWalletCore is Initializable, UUPSUpgradeable, OwnableUpgradeable {
             } catch Error(string memory reason) {
                 success = false;
                 returnData = bytes(reason);
-            } catch {
-                success = false;
-                returnData = bytes("err executing extension");
+            } catch (bytes memory lowLevelData) {
```

Comment:
@SoraSuegami 
With the use of contract in v5, return data is now Custom Error, so I am unable to get the reason and have a selector as shown below. I would like to discuss the implementation policy, whether to implement all types of errrors or ignore the reason strings.